pmlc/dialect/tile/tests/expand_reshape.mlir

```diff
@@ -0,0 +1,57 @@
+// RUN: pmlc-opt -tile-expand-reshape %s | FileCheck %s
+
+// CHECK: #[[$map0:.*]] = affine_map<(d0, d1, d2, d3) -> (0, d0, d1, d3 + d2 * 4)>
```

Comment:
nit: we usually don't add a `$` prefix to regex names

Reply:
I tried and it didn't work without '$'

Reply:
OK, thanks